spaceship/spaceship.gemspec

```diff
@@ -31,7 +31,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'faraday', '~> 0.9'
   spec.add_dependency 'faraday_middleware', '~> 0.9'
   spec.add_dependency 'faraday-cookie_jar', '~> 0.0.6'
-  spec.add_dependency 'fastimage', '~> 1.6'
```

Comment:
Do we want to provide a minimum version `>= 1.6` to make sure the user doesn't use a really old version?